src/main/java/uk/gov/hmcts/reform/bulkscan.payment.processor/config/QueueClientsConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.config;
+
+import com.microsoft.azure.servicebus.ClientFactory;
+import com.microsoft.azure.servicebus.IMessageReceiver;
+import com.microsoft.azure.servicebus.ReceiveMode;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class QueueClientsConfiguration {
+
+    @Bean("payments-messageReceiver")
```

Comment:
How about `payments-message-receiver` or `payments-receiver`? If we have dashes as word separators, it would be best to do all lowercase.

Reply:
both fine for me , I just want to add the type info to name.
may be  we do not  need any name,

Reply:
You're right, if there's a message receiver for just one queue, there is no such need. If another queue appeared at some point (which we're not planning), we can add names at that point.

Reply:
remove the name